python/lsst/rubintv/production/processingControl.py

```diff
@@ -0,0 +1,1042 @@
+                can be called.
+            setterPart : `str`
+                If a setter type command, what the component is being set to,
+                such that it can be instantiated.
+            """
+            getterParts = None
+            setterPart = None
+            if "=" in command:
+                getterPart, setterPart = command.split("=")
+                getterParts = getterPart.split(".")
+            else:
+                getterParts = command.split(".")
+            return getterParts, setterPart
+
+        def safeEval(setterPart):
+            """Ensure whatever we're being asked to instantiate is safe to.
+
+            If a primative is passed, it's safely evaluated with literal_eval,
+            otherwise, it's only instantiated if it's already an item in the
+            global namespace, ensuring that arbitrary code execution cannot
+            occur.
+
+            Parameters
+            ----------
+            setterPart : `str`
+                Whatever item we need to instantiate.
+
+            Returns
+            -------
+            item : `obj`
+                Whatever item was asked for.
+
+            Raises
+                ValueError if the item could not be safely evaluated.
+            """
+            try:
+                # if we have a primative, get it simply and safely
+                item = literal_eval(setterPart)
+                return item
+            except (SyntaxError, ValueError):  # anything non-primative will raise like this
+                pass
+
+            if setterPart.split(".")[0] in globals():
+                # we're instantiating a known class, so it's safe
+                item = eval(setterPart)
+                return item
+            raise ValueError(f"Will not execute arbitrary code - got {setterPart=}")
+
+        # command list is a list of dict: dict with each dict only having a
+        # single key, and the value being the kwargs, if any.
+        for command in commandList:
+            try:
+                for method, kwargs in command.items():
+                    getterParts, setter = parseCommand(method)
+                    component = getBottomComponent(parentClass, getterParts[:-1])
+                    functionName = getterParts[-1]
+                    if setter is not None:
+                        setItem = safeEval(setter)
+                        component.__setattr__(functionName, setItem)
+                    else:
+                        attr = getattr(component, functionName)
+                        attr.__call__(**kwargs)
+            except Exception as e:
+                self.log.exception(f"Failed to apply command {command}: {e}")
+                return  # do not apply further commands as soon as one fails
+
+
+class CameraControlConfig:
+    """Processing control for which CCDs will be processed."""
+
+    def __init__(self):
+        self.camera = LsstCam.getCamera()
```

Comment:
OK, I think that's fair. I'm filing a new ticket for that though, to unblock the merge, but I think it might just trivially work for all cameras actually, now that I look at it, I just don't have time before OR4 to test that properly.